articles/generative_ai/data_extraction_from_social_media_posts_using_llms/README.md

```diff
@@ -0,0 +1,55 @@
+# Data extraction from Social Media posts using LLM's
+
+This course covers the implementation of an application, capable of extracting social media posts from instagram profiles 
+and creating a weekly report based on them, using the power of LLM's.
+
+![Thumbnail Image](./media/article_cover.jpg)
+
+## Table of Contents
+
+- [Articles](#article)
+- [Dependencies](#dependencies)
+- [Install](#install)
+- [Usage](#usage)
+- [License](#license)
+
+
+------
+
+## Article
+
+To fully grasp the code, check out our articles ↓
+
+[Full Article](https://medium.com/p/6ebe0e2cdeb1/edit)
+
+## Dependencies
+
+- [Python (version 3.11)](https://www.python.org/downloads/)
+- [Poetry (version 1.6.1)](https://python-poetry.org/)
+- [GNU Make (version 3.81)](https://www.gnu.org/software/make/)
+- [Docker (version 24.0.7)](https://www.docker.com/)
+
+## Install
+
+As we use Make and Poetry to manage the project, to install the project, you have to run the following:
+```shell
+make setup
+```
+
+
+## Usage
+
+To start the hole process you have to run the following: 
+
+```shell
+make run
+```
```

Comment:
Could you also add a sentence about on the env vars you have to fill out?

Something like: `cp .env.example .env` and fill out the env vars from there. We have to assume people don't know how to use `.env` files (we are working with many data scientists 🧑‍🔬)